python/cudf/cudf/core/column/numerical.py

```diff
@@ -333,15 +333,17 @@ def find_first_value(self, value, closest=False):
         """
         found = 0
         if len(self):
-            found = cudautils.find_first(self.data_array_view, value)
+            found = cudautils.find_first(
+                self.data_array_view, value, mask=self.mask
+            )
```

Comment:
I'm not seeing the changes to this function to take in the mask, can you point them out?

Reply:
`self.data_array_view, value, mask=self.mask`

I am using the mask directly on the index column, so when I search for max or min I don't have worry.

Reply:
And I had missed to push the cudautils.py which  had the change, my mistake.